src/System.Runtime.Extensions/tests/System/Convert.ToBoolean.cs

```diff
@@ -124,5 +123,20 @@ public void FromUInt64()
             Boolean[] expectedValues = { false, true };
             Verify(Convert.ToBoolean, testValues, expectedValues);
         }
+#if netstandard17
```

Comment:
Nit: you are also missing a new line before this #if's in some of your test files.